packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -429,17 +429,17 @@ export const transformPlotlyJsonToSankeyProps = (
     }),
   };
 
-  const width: number = layout?.width || 440;
-  const height: number = layout?.height || 220;
+  const width: number = typeof layout?.width === 'number' ? layout?.width : 440;
+  const height: number = typeof layout?.height === 'number' ? layout?.height : 220;
   const styles: ISankeyChartProps['styles'] = {
     root: {
-      fontSize: layout.font?.size,
+      fontSize: typeof layout.font?.size === 'number' ? layout.font?.size : 0,
```

Comment:
dont use 0 font size

Reply:
use the default font size used for donut text

Reply:
arcLabel uses a token. Then don't set this style override at all if font size is not defined or non number

Reply:
This is true for all charts? if fontsize is not defined or non-number, then we will not define this property.

Reply:
leave title and legend values. They need more handling. For others default value should be defined already.